anybox/recipe/odoo/runtime/session.py

```diff
@@ -330,22 +350,28 @@ def update_modules_list(self):
 
         This is necessary prior to install of any new module.
         """
-        self.registry('ir.module.module').update_list(self.cr, self.uid)
+        if hasattr(odoo, 'api'):
+            self.env['ir.module.module'].update_list()
+        else:
+            self.registry('ir.module.module').update_list(self.cr, self.uid)
 
     def init_cursor(self):
         db = getattr(self._registry, 'db', None)
         if db is None:  # current trunk (future v8)
             self.cr = self._registry.cursor()
         else:
-            # In OpenERP < 8, Registry.cursor() object is
+            # In Odoo < 8, Registry.cursor() object is
             # a context manager providing auto closing,
             # but we don't want to control the whole lifespan
             # of the cursor.
             self.cr = db.cursor()
 
     def registry(self, model):
         """Lookup model by name and return a ready-to-work instance."""
-        return self._registry.get(model)
+        if hasattr(odoo, 'api'):
+            return self.env[model]
+        else:
+            return self._registry.get(model)
```

Comment:
I've just read your commit message it's about the release process, so I'll keep that one in mind!